bin/ebs-autoscale

```diff
@@ -180,8 +185,11 @@ while true; do
     # get number of devices only when we need to add more space
   	NUM_DEVICES=$(get_num_devices)
     logthis "LOW DISK (${PCT_UTILIZATION}%): Adding more."
-    add_space "$NUM_DEVICES"
-	NUM_DEVICES=$(expr $NUM_DEVICES + 1 )
+    if ! add_space "$NUM_DEVICES"; then
+      logthis "Failed to add space, retrying"
```

Comment:
are we retrying though? you have added a break

Reply:
good catch! should be `continue`.